app/views/talks/_card.html.erb

```diff
@@ -2,7 +2,14 @@
 <%# locals: (talk:, user_favorite_talks_ids: [], favoritable: false, back_to: nil, back_to_title: nil) -%>
 
 <div class="card card-compact bg-white shadow-xl w-full" id="<%= dom_id talk %>" style="<%= "view-transition-name: #{dom_id(talk, :talk_card)}" %>">
-  <%= link_to talk_path(talk, back_to: back_to, back_to_title: back_to_title), class: "flex aspect-video overflow-hidden" do %>
+  <%= link_to talk_path(talk, back_to: back_to, back_to_title: back_to_title), class: "flex aspect-video overflow-hidden relative" do %>
+    <% if talk.static_metadata.language.present? && talk.static_metadata.language.downcase != "english" %>
```

Comment:
Talk model has a langage attribute, any reason why we are not using it here?

Reply:
hmm, there's was a reason why I didn't, but I'm blanking on why now. Let me change it 👍🏼

Reply:
oh, I guess that's why...
![CleanShot 2024-10-20 at 22 13 36](https://github.com/user-attachments/assets/dba92376-2469-4591-883c-76f03d621caa)

This doesn't make any sense

Reply:
Updated 👍🏼

Reply:
I don't really like this with_essential_card_data but it was the only way to get decent perf for topic.talks very strange